site/content/en/docs/handbook/controls.md

```diff
@@ -5,7 +5,9 @@ weight: 2
 description: >
   See minikube in action!
 aliases:
-  - /docs/examples/
+
```

Comment:
I don't think we need a newline here

Reply:
site/content/en/docs/handbook/controls.md:9 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "- /docs/examples/"]